server/devdocs/bin/generate-proptypes-index.js

```diff
@@ -83,10 +176,10 @@ const parseDocument = ( docObj ) => {
 
 /**
  * Creates an index of the files
- * @param {Array} parsed
- * @return {{data: Array, index: {displayName: {}, slug: {}, includePath: {}}}}
+ * @param {Array} parsed The parsed documentation array
+ * @return {Array} Removes any components without a displayName
  */
-const createIndex = ( parsed ) => {
+const cleanIndex = ( parsed ) => {
```

Comment:
You make a very good point, though IIRC, this function once had much more stuff in it and was cut down to just this singular filter...